packages/e2e-tests/helpers/index.ts

```diff
@@ -77,6 +83,12 @@ export async function startLibraryTest(page: Page, apiKey: string, teamId: strin
   await page.locator('[data-test-id="TestRunList"]').waitFor();
 }
 
+export async function commandPalette(page: Page, query: string) {
+  await page.keyboard.press(`${getCommandKey()}+K`);
+  await page.keyboard.type(query);
```

Comment:
I'm surprised we don't need a wait-for in between these two (for the type ahead to become visible). I assumed that's what you added the new `command-palette` test id for, e.g.

```js
await page.waitForSelector('[data-test-id="command-palette"]`);
```

Reply:
Same. I like that Playwright is flexible though

Reply:
I guess we'll see if this ends up being a source of flakiness. (Maybe the default interaction time is just enough for React to render the type ahead)